contrib/pyln-testing/pyln/testing/fixtures.py

```diff
@@ -127,10 +127,21 @@ def bitcoind(directory, teardown_checks):
 
     info = bitcoind.rpc.getnetworkinfo()
 
-    if info['version'] < 160000:
+    # FIXME: include liquid-regtest in this check after elementsd has been
+    # updated
+    if info['version'] < 200100 and env('TEST_NETWORK') != 'liquid-regtest':
         bitcoind.rpc.stop()
-        raise ValueError("bitcoind is too old. At least version 16000 (v0.16.0)"
+        raise ValueError("bitcoind is too old. At least version 20100 (v0.20.1)"
                          " is needed, current version is {}".format(info['version']))
+    elif info['version'] < 160000:
+        bitcoind.rpc.stop()
+        raise ValueError("elementsd is too old. At least version 160000 (v0.16.0)"
+                         " is needed, current version is {}".format(info['version']))
+
+    # At 0.21, createwallet needed, but in 0.20.1 createwallet then broke
+    # calls to getnewaddress with "bitcoin.rpc.JSONRPCError: {'code': -19, 'message': 'Wallet file not specified (must request wallet RPC through /wallet/<filename> uri-path).'}"
+    if info['version'] > 200100:
+        bitcoind.rpc.createwallet("test")
```

Comment:
This is exactly how I tried to fix it first in https://github.com/ElementsProject/lightning/pull/4088, but in the end I used the `-wallet="default"` option in `contrib/pyln-testing/pyln/testing/utils.py` because it is backwards compatible with at least 0.20.1. @rustyrussell please consider which should stay.